### PR TITLE
DT-571: Fix database properties and more testing

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: src/main/resources/db/migration/V1_2__insert_prison_gp_practice.sql [hunk 1]
```diff
@@ -53,7 +53,6 @@ INSERT INTO prison_gp_practice VALUES ('HMI', 'Y05528');
 INSERT INTO prison_gp_practice VALUES ('HCI', 'Y03223');
 INSERT INTO prison_gp_practice VALUES ('ISI', 'Y05765');
 INSERT INTO prison_gp_practice VALUES ('IWI', 'Y05532');
-INSERT INTO prison_gp_practice VALUES ('IWI', 'Y05534');
```
Comment (petergphillips) [inline src/main/resources/db/migration/V1_2__insert_prison_gp_practice.sql:56]: duplicate row for `IWI` - temporarily take out until work out best way to resolve.

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonestate/jpa/PrisonRepositoryTest.kt [hunk 2]
```diff
@@ -46,6 +46,17 @@ class PrisonRepositoryTest {
...
     assertThat(prison.gpPractice).isEqualTo(PrisonGpPractice("MDI", "Y05537"))
+  }
+
+  @Test
+  fun `should get active prisons`() {
+    val activePrisons = prisonRepository.findByActiveOrderByPrisonId(true)
+    assertThat(activePrisons).hasSizeGreaterThan(100)
+
+    val inActivePrisons = prisonRepository.findByActiveOrderByPrisonId(false)
+    assertThat(inActivePrisons).hasSizeGreaterThan(40)
```
Comment (andymarke) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/prisonestate/jpa/PrisonRepositoryTest.kt:57]: maybe also add `hasSizeLessThan(100)` to clarify you get less than the active ones

**Review by andymarke — APPROVED** [review]
LTGM

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonestate/jpa/PrisonRepositoryTest.kt (hunk 2 above)
Comment (petergphillips) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/prisonestate/jpa/PrisonRepositoryTest.kt:57]: I'll add
```kotlin
    assertThat(inActivePrisons).hasSizeGreaterThan(40).allMatch { !it.active }
```
into my next PR

**Pull request merged**